head.html

```diff
@@ -5,6 +5,7 @@
     <link rel="canonical" href="https://{{host}}/">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     
+    <script defer type="application/javascript" src="/scripts/redirect.js"></script>
```

Comment:
```suggestion
    <script defer type="text/javascript" src="/scripts/redirect.js"></script>
```
From RFC 9239:
> The media type registrations herein are divided into two major categories: (1) the sole media type
"text/javascript", which is now in common usage and (2) all of the media types that are obsolete
(i.e., "application/ecmascript", "application/javascript", "application/x-ecmascript", "application/
x-javascript", "text/ecmascript", "text/javascript1.0", "text/javascript1.1", "text/javascript1.2", "text/
javascript1.3", "text/javascript1.4", "text/javascript1.5", "text/jscript", "text/livescript", and "text/xecmascript").